src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/WebservicePage/webservice.html.twig

```diff
@@ -0,0 +1,81 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Admin.Advparameters.Feature" %}
+
+{% block content %}
+    {% block logs_main_panel %}
+        <div class="col-md-12">
+            <div class="col">
+                {{ include('@PrestaShop/Admin/Common/Grid/grid_panel.html.twig', {'grid': grid }) }}
+            </div>
+        </div>
+    {% endblock %}
+
+    {% block webservice_configuration_panel %}
+    <div class="col-md-12">
+        {{ form_start(form, {'attr' : {'class': 'form'} }) }}
+        <div class="row">
+            {% block webservice_configuration %}
+                <div class="col">
+                    <div class="card">
+                        <h3 class="card-header">
+                            <i class="material-icons">settings</i> {{ 'Configuration'|trans }}
+                        </h3>
+                        <div class="card-block">
+                            <div class="card-text">
+                                <div class="form-group">
+                                    <label class="form-control-label">{{ 'Enable PrestaShop\'s webservice'|trans }}</label>
+                                    {{ form_errors(form.webservice_configuration.enable_webservice) }}
+                                    {{ form_widget(form.webservice_configuration.enable_webservice) }}
+                                </div>
+
+                                <small class="form-text">{{ 'Before activating the webservice, you must be sure to:\n 1. Check that URL rewriting is available on this server.\n 2.Check that the five methods GET, POST, PUT, DELETE and HEAD are supported by this server.'|trans({}, 'Admin.Shopparameters.Help')|nl2br }}</small>
```

Comment:
Is the whole string wrote properly? I have three separate strings in Crowdin.

Reply:
In _Admin/Advparameters.Help_ !

Reply:
you're right @LouiseBonnard.

Please @matks be careful with translation keys, if we change them we lost all related translations.

Reply:
@mickaelandrieu Note to myself: add this to the Migration Guide